doc/release-notes-13756.md

```diff
@@ -0,0 +1,39 @@
+Coin selection
+--------------
+
+### Reuse Avoidance
+
+A new wallet flag `avoid_reuse` has been added (default off). When enabled,
+a wallet will distinguish between used and unused addresses, and default to not
+use the former in coin selection.
+
+(Note: rescanning the blockchain is required, to correctly mark previously
```

Comment:
Could remove `(Note: `, since all sentences in this doc are release *notes*

Reply:
Removed. I wanted to use it as in "Do note that [word of caution]", but it's no big deal.